cmake/modules/curl.cmake

```diff
@@ -73,6 +73,7 @@ else()
 			--without-libpsl
 			--without-nghttp2
 			--without-libssh2
+			--with-ca-path=/etc/ssl/certs/
```

Comment:
@poiana This option only fixes the problem for distros that use the certificate directory format where each root CA gets its own entry in the `/etc/ssl/certs` directory.

RHEL-based distros do not use those. They instead put them at `/etc/ssl/certs/ca-bundle.crt` (ubuntu actually instead uses `/etc/ssl/certs/ca-certificates.crt`, so there is no single file that is common across both).

That said, I _think_ that if you specify both:
```
--with-ca-bundle=/etc/ssl/certs/ca-bundle.crt
--with-ca-path=/etc/ssl/certs
```
then it should work on both ubuntu and rhel-based distros. 

There is also a chance that if you simply install the `ca-certificates` package in the build environment and build the bundled openssl pointing at the correct ssl dir for the build environment that you can omit this setting here, and it will use defaults that work on any distro at runtime, but I'm not sure about that.

Also, this parameter is not supposed to have a trailing `/`. If you look in strace, that results in paths with a double `/` in them like:
```
[pid 3558894] newfstatat(AT_FDCWD, "/etc/ssl/certs//52efdea2.0", 0xffffb33d7b30, 0) = -1 ENOENT (No such file or directory)
```